src/Module/Moderation/Report/Create.php

```diff
@@ -280,6 +280,8 @@ private function summary(array $request): string
 			'$block'    => ['contact_action', $this->t('Block contact'), self::CONTACT_ACTION_BLOCK, $this->t("Their posts won't appear in your Network page anymore, but their replies can appear in forum threads, with their content collapsed by default. They cannot follow you but still can have access to your public posts by other means.")],
 
 			'$display_forward' => !$this->baseUrl->isLocalUrl($contact['url']),
+			'$forward'         => ['report_forward', $this->t('Forward report'), self::CONTACT_ACTION_BLOCK, $this->t('Would you like to forward this report to the remote server?')],
+			// @deprecated 2025.04 remove the next line after the above line is translated.
 			'$forward'         => ['report_forward', $this->t('Forward report'), self::CONTACT_ACTION_BLOCK, $this->t('Would you ike to forward this report to the remote server?')],
```

Comment:
Please move this 3 lines over the array to an extra variable, or PHPStan will warn about the duplicate array key.